autopush/webpush_server.py

```diff
@@ -380,13 +382,13 @@ def lookup_user(self, hello):
         # All records must have a router_type and connected_at, in some odd
         # cases a record exists for some users without it
         if "router_type" not in record or "connected_at" not in record:
-            self.db.router.drop_user(uaid)
+            self.drop_user(uaid, record, 104)
```

Comment:
We send the entire uaid record out to logging?

Reply:
yea, this is what websocket and apparently web.webpush also do. doesn't seem necessary but at least they're all log.debug's